pkg/cmd/run.go

```diff
@@ -137,14 +137,22 @@ doppler run --mount secrets.json -- cat secrets.json`,
 		dopplerSecrets := fetchSecrets(localConfig, enableCache, fallbackOpts, metadataPath, nameTransformer, dynamicSecretsTTL)
 
 		mountPath := cmd.Flag("mount").Value.String()
-		mountFormat := cmd.Flag("mount-format").Value.String()
+		mountFormatString := cmd.Flag("mount-format").Value.String()
 		mountTemplate := cmd.Flag("mount-template").Value.String()
 		maxReads := utils.GetIntFlag(cmd, "mount-max-reads", 32)
 		// only auto-detect the format if it hasn't been explicitly specified
 		shouldAutoDetectFormat := !cmd.Flags().Changed("mount-format")
 		shouldMountFile := mountPath != ""
 		shouldMountTemplate := mountTemplate != ""
 
+		var mountFormat *models.SecretsMountFormat
+		if mountFormatString != "" {
+			mountFormat = models.SecretsMountFormatMap[mountFormatString]
+			if mountFormat == nil {
+				utils.HandleError(fmt.Errorf("invalid mount format. Valid formats are %s", validMountFormatsList))
```

Comment:
In `MountSecrets()` we use a slightly different error message. Can we use the same error in both places? I don't have a preference for which version we use, though you'll want to capitalize `invalid` to `Invalid`.

In `MountSecrets()` we use:
```
fmt.Errorf("Invalid mount format: %s", format)
```

Reply:
Done!